src/BlockTypes/ClassicTemplate.php

```diff
@@ -61,7 +61,7 @@ protected function render( $attributes, $content, $block ) {
 			$frontend_scripts::load_scripts();
 		}
 
-		$archive_templates = array( 'archive-product', 'taxonomy-product_cat', 'taxonomy-product_tag', ProductSearchResultsTemplate::SLUG );
+		$archive_templates = array( 'archive-product', 'taxonomy-product_cat', 'taxonomy-product_tag', 'taxonomy-pa', ProductSearchResultsTemplate::SLUG );
```

Comment:
I think that it makes sense to use the SLUG constant.

```suggestion
		$archive_templates = array( 'archive-product', 'taxonomy-product_cat', 'taxonomy-product_tag', 'ProductAttributeTemplate::SLUG', ProductSearchResultsTemplate::SLUG );
```

Don't forget to import the class in this file!